src/app/home/home.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="container">
+  <div class="header">
+    <h1>TYPE SOME INFO ABOUT YOU</h1>
+  </div>
+  <form #f="ngForm" (ngSubmit)="onSubmit(f)">
+    <div class="name">
+      <mat-form-field appearance="standard">
+        <mat-label>Name</mat-label>
+        <input matInput ngModel name="name" required />
+      </mat-form-field>
+    </div>
+
+    <div>
+      <mat-form-field appearance="standard">
+        <mat-label>Surname</mat-label>
+        <input matInput ngModel name="surname" required />
+      </mat-form-field>
+    </div>
+
+    <div>
+      <mat-form-field class="example-full-width">
+        <mat-label>Date of birth</mat-label>
+        <input type="date" matInput ngModel name="date" required />
```

Comment:
I would rename it to something more meaningful like `birthDate`. We have a label here so we kind of know what to expect, but we're missing context in `.ts` files